recipes/metaknowledge/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "metaknowledge" %}
+{% set version = "3.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f137dce7ed7e1fb3a73730716d37e2b5757d9fc94f3bb50f8e82338c38b17dae
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - networkx
+
+test:
+  imports:
+    - metaknowledge
+
+about:
+  home: https://github.com/networks-lab/metaknowledge
+  license: GPL2
+  license_family: GPL
+  license_file: LICENSE
```

Comment:
Your license is not present with the source code
You can add it manually

Reply:
Ah, OK, sorry, I didn't realize the authors of the package didn't include it. I just added the license file.